iocore/net/QUICNetVConnection.cc

```diff
@@ -713,7 +807,9 @@ QUICNetVConnection::registerNextProtocolSet(SSLNextProtocolSet *s)
 bool
 QUICNetVConnection::is_closed()
 {
-  return this->handler == reinterpret_cast<NetVConnHandler>(&QUICNetVConnection::state_connection_closed);
+  return (this->handler == reinterpret_cast<NetVConnHandler>(&QUICNetVConnection::state_connection_closed) ||
```

Comment:
This might be a bit confusing. If you really want to check this state with one function, you should probably introduce another function for it.

Reply:
I just want test whether a qvc is closed or not. That means a qvc can't be used any more if we at these states.

But I'm ok to introduce another function.